content/posts/2016-03-22-what-was-your-path-to-18f.md

```diff
@@ -143,7 +143,7 @@ _The acquisitions team provides innovative services and solutions to support dig
   “Lots of time in the non-profit world with a focus on setting up systems for people to do their jobs better — not realizing until fairly recently there were entire professions of the skills I was using (user research, user experience, information architecture, curriculum development). After meeting 18F-ers I realized there were big teams of people who loved helping other people do their work better. <3”  - {{ "jessie-posilkin" | team_link }}
   </blockquote>
 </div>
-<div class="pquote">{{ "rebecca-refoy-sidibe" | team_photo }}
```

Comment:
clarification: I'm not seeing the image file renamed in this PR — have you checked to make sure this image is rendering correctly?

Reply:
There is no photo for the original author slug in the `assets/img/team` directory, so `team_photo` displays an 18F logo. That behavior is visible in production: https://18f.gsa.gov/2016/03/22/what-was-your-path-to-18f/

The change to the slug here doesn't change any behavior but keeps the author slug naming consistent.